neovim/ui/gtk_ui.py

```diff
@@ -152,7 +152,11 @@ def _nvim_resize(self, columns, rows):
         self._cell_pixel_width = cell_pixel_width
         self._cell_pixel_height = cell_pixel_height
         self._screen = Screen(columns, rows)
-        self._window.resize(pixel_width, pixel_height)
+        prev_rect = self._drawing_area.get_allocation()
```

Comment:
> If we were to develop this gui futher, I think we eventually should split the gui out from the python client/host repo, as one might want to fork/experiment with the gui, but still follow the upstream python-client code. Ping @tarruda

:+1: 

Ideally we'd have a neovim gtk widget in C so it can be used by any language with gtk bindings